mitosheet/src/components/taskpanes/UpdateImports/UpdateImportsPreReplayTaskpane.tsx

```diff
@@ -63,10 +67,12 @@ const getErrorMessage = (
 
     let errorText: string | undefined = undefined;
 
-    if (numImportsInvalidPreReplayThatAreNotUpdated > 0 || numImportsInvalidPostReplay > 0) {
-        errorText = 'Please fix the failed imports below before updating imports';
-    } else if (invalidReplayError !== undefined) {
-        errorText = invalidReplayError;
+    if (!loadingImportDataAndErrors) {
+        if (numImportsInvalidPreReplayThatAreNotUpdated > 0 || numImportsInvalidPostReplay > 0) {
```

Comment:
`numImportsInvalidPreReplayThatAreNotUpdated` would this just be `numImportsInvalidPreReplay`?
What is the note supported part?

Reply:
No - this is the number of imports that were invalid before the replay occurred, that are still not updated!